### PR TITLE
Moved backOff and limitter logic to SDK side

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

  # To Test

1. Quote API (Not enough liquidity and order errors)
2. Order placement
3. Order cancellation
3. Orders list loading
4. Should inherit the behaviour from https://github.com/cowprotocol/cowswap/pull/2069

### CODE REVIEW

File: src/cow-react/api/gnosisProtocol/api.ts [hunk 1]
```diff
@@ -72,7 +65,8 @@ function _fetchProfile(
   data?: any
 ): Promise<Response> {
   const baseUrl = _getProfileApiBaseUrl(chainId)
-  return fetchRateLimitted(baseUrl + url, {
+
+  return fetch(baseUrl + url, {
```
Comment (shoom3301) [inline src/cow-react/api/gnosisProtocol/api.ts:69]: Actually, we even don't use the method anymore, should be deleted at all

**Review by anxolin — APPROVED** [review]
Nice!

**Review by anxolin — COMMENTED** [review]
@shoom3301 shouldn't we use the backoff and ratelimit from the SDK for other things? (since is a C&P?

We can import from SDK, right?

**Comment by shoom3301** [comment]
@anxolin I don't think so. They are external dependencies (`exponential-backoff` and `limiter`). And it will be not so clear, if we use them from `@cowprotocol/cow-sdk`

**Comment by elena-zh** [comment]
Hey @shoom3301 , nice!
However, I noticed that something wrong with a hard cancellation in the PR: the order shows that it was cancelled in the CoW Swap, but order details in Explorer show that the order is in [Open ](https://dev.explorer.cow.fi/goerli/orders/0xd7466ee134066ef851ea2aae0bccf7aa3c5e187cf4937f3e0b0ab3de2135d4129fa3c00a92ec5f96b1ad2527ab41b3932efeda5864490f59?tab=overview)status (and order requests in the console show 404 error). 
open
image
image
image


In fact, the order is executed (I see it based on the balances in my wallet)


Then, I have placed 12 orders, and can see 503 errors in the console 
image
However, not sure if it is related to the current PR as there is a warning 
image

Thanks!

**Comment by shoom3301** [comment]
@elena-zh hi! These issues are not related to current PR

**Comment by elena-zh** [comment]
Hey @shoom3301 , indeed. I have been able to catch the 1st issue on [Dev today.](https://cowservices.slack.com/archives/C0361CDG8GP/p1682595815775429) 
The 2nd was related to another issue as I supposed to. 
The current PR is approved.

**Review by elena-zh — APPROVED** [review]

**Pull request merged**